pkg/frontend/admin_openshiftcluster_kubernetespods_logs.go

```diff
@@ -0,0 +1,92 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+)
+
+func (f *frontend) getAdminKubernetesPodLogs(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
+	b, err := f._getAdminKubernetesPodLogs(ctx, r, log)
+
+	adminReply(log, w, nil, b, err)
+}
+
+func (f *frontend) _getAdminKubernetesPodLogs(ctx context.Context, r *http.Request, log *logrus.Entry) ([]byte, error) {
+	vars := mux.Vars(r)
+
+	containerName, namespace, name := r.URL.Query().Get("container"), r.URL.Query().Get("namespace"), r.URL.Query().Get("name")
+
+	err := validateAdminKubernetesPodLogs(name, namespace, containerName)
+	if err != nil {
+		return nil, err
+	}
+
+	resourceID := strings.TrimPrefix(filepath.Dir(r.URL.Path), "/admin")
+
+	doc, err := f.dbOpenShiftClusters.Get(ctx, resourceID)
+	switch {
+	case cosmosdb.IsErrorStatusCode(err, http.StatusNotFound):
+		return nil, api.NewCloudError(http.StatusNotFound, api.CloudErrorCodeResourceNotFound, "", "The Resource '%s/%s' under resource group '%s' was not found.", vars["resourceType"], vars["resourceName"], vars["resourceGroupName"])
+	case err != nil:
+		return nil, err
+	}
+
+	k, err := f.kubeActionsFactory(log, f.env, doc.OpenShiftCluster)
+	if err != nil {
+		return nil, err
+	}
+
+	kubeInterface, err := f.getKubernetesClientSet(ctx, r, log)
+	if err != nil {
+		return nil, err
+	}
+
+	podLogs, err := k.KubeGetPodLogs(ctx, namespace, name, containerName, kubeInterface)
+	if err != nil {
+		return nil, err
+	}
+
+	return podLogs, nil
```

Comment:
```suggestion
return k.KubeGetPodLogs(ctx, namespace, name, containerName, kubeInterface)

```
we can just return this function call